test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/common.lua

```diff
@@ -83,6 +83,29 @@ function m.getVehicleData(pData, pParam)
     m.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS", [pParam] = pData })
 end
 
+function m.getVehicleDataWithOneParam(pData)
+  local cid = m.getMobileSession():SendRPC("GetVehicleData", { fuelRange = true })
```

Comment:
@dboltovskyi 
I'm sorry. This function in the common file is not needed and was removed. Local functions have been added to 002 and 006 scripts.